### ISSUE TITLE
How to install, given new ownership?

### ISSUE THREAD

**Issue by PeteMichaud** [obsidian-community/obsidian-pandoc#2]
The fork here is a little confusing. I understand why it exists, but is this codebase now where the Community plugins UI is pulling from, or do I need to manually install this version?

I think if this switch is happening, it should probably fully happen, ie. the community plugins should point to this repo, the readme links should be updated to point here instead of at the old repo, etc. 

If this is welcome I can probably do this myself and submit a PR, but I want to coordinate about it first.

**Comment by AB1908** [comment]
Yep, this question was going to pop up soon. We need to migrate the issues and maybe create a sort of GitHub action to build the release. I don't think there'll be a clear way to point from that repo to this but I may be wrong.

**Comment by Limezy** [comment]
Could you maybe clarify the status of that fork in the readme or in a discord post ? Is this fork a personal thing, or a collective action  ? (I think I remember 6 months ago a group of active users waiting for Olivier Balfour'a answer about his plugin). Should we copy paste the pending issues from the original repo ? Should we amend Obsidian's plugin list to point out to that new and updated repo ?... 
Anyway thanks because I believe Obsidian pandoc really needs some updates and it's an important plugin

**Comment by AB1908** [comment]
You're right. I'll try and prioritize that status and how-to at the very least. Issue migration and other updates will take a bit longer.

**Comment by ReaderGuy42** [comment]
Hi guys, has there been an update on this? Just curious :) Thanks!

**Comment by AB1908** [comment]
Sorry, this has fallen down the pecking order. Currently, this can be installed via the BRAT plugin I believe.

**Comment by WingerathM** [comment]
Can you maybe clarify on the BRAT installation process? If I paste the repo's URL in BRAT I get an error message, saying the file "main.js" is missing. As far as I can tell, the "main.js" will be included in a release version (aka will be build for that release?). I'm fairly new to using github and obsidian. Is there a way to "build" the release on my own? The BRAT process worked for the fork of Phillip Johnston (https://github.com/phillipjohnston/obsidian-pandoc), which has a release.

**Comment by AB1908** [comment]
You're right, this repo doesn't have a release. Continue to use Philip's fork but let me try setting up a GH action to automatically build a release. 

On a separate note, @phillipjohnston, do you think you we can merge some of your improvements and push out a release?

**Comment by AB1908** [comment]
I'll actually go ahead and create a release manually for now.

**Comment by phillipjohnston** [comment]
@AB1908 changes were merged in with a PR already.

**Comment by AB1908** [comment]
Ah, I guess I should've checked. Thanks!
